sockutils.c

```diff
@@ -65,6 +65,10 @@
 #include "portability.h"
 #include "sockutils.h"
 
+#if defined(_MSC_VER) && !defined(strdup)
```

Comment:
And again - and `portability.h` is directly included *right above this*.

Reply:
Yes, and yet, without it:
`4>C:\Users\blazeable\GitHub\libpcap\sockutils.c(889): warning C4013: 'strdup' undefined; assuming extern returning int`

Reply:
Again, why must this be handled in several places in the source rather than by a change to `portability.h`?